test/db_test.py

```diff
@@ -61,6 +61,21 @@ def noUserFound():
 			user = User.get_by_name('badname')
 			assert_that(user, is_(none()))
 
+		@it('User accessed_at field updates (And nothing else)')
+		def userAccessedAt():
+			before = User.get_by_name(test_name).__dict__.get('__data__')
+			after = User.get_by_name(test_name).__dict__.get('__data__')
+
+			'''This works because the date is stored in nanoseconds.'''
+			assert_that(
+				before.pop('accessed_at'),
```

Comment:
Kind of not a fan of modifying the data while in the middle of performing an assertion on it. Maybe we can directly check `accessed_at` here, then call `model_to_dict` right before the next assertion? 

This is a little outside the scope of this task, but we could also write a custom matcher for PyHamcrest that works with peewee models. Something like
`assert_that(before, matches_model(after).excluding_fields('accessed_at'))`